### ISSUE TITLE
Error when setting policy to bucket in helm chart

### ISSUE THREAD

**Issue by elg0ch0** [minio/minio#16046]
When specifying a bucket in the values.yaml file:
```
   - name: bucket001
     policy: download
     purge: false
     versioning: true
     objectlocking: false

```

the policy is not assigned to the bucket.
The bucket job's log shows:

```
Setting policy of bucket 'bucket001' to 'download'.
mc: Please use 'mc anonymous'

```

which makes me think the creation function is missing a parameter (**anonymous** ?)

## Expected Behavior
Bucket is created with the specified policy (download in the above example)

## Current Behavior
Bucket is created without the specified policy (download in the above example)

## Steps to Reproduce (for bugs)
1. Provide a values file with a bucket to be created setting a policy:
```
   - name: bucket001
     policy: download
     purge: false
     versioning: true
     objectlocking: false
```

## Context
N/A

## Your Environment

* Version used (`minio --version`): RELEASE.2022-11-07T23-47-39Z
* Server setup and configuration:
* Operating System and version (`uname -a`): Ubuntu 20

**Comment by harshavardhana** [comment]
Feel free to send a fix @elg0ch0

**Comment by harshavardhana** [comment]
Fixed via #16055
